zipline/pipeline/factors/statistical.py

```diff
@@ -0,0 +1,270 @@
+
```

Comment:
We should probably move the base correlation and regression factors here as well.  (I think that means you'll need to do a lazy import in the factor methods that emit those types, which is okay here.)